<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hpmainUi</class>
 <widget class="QWidget" name="hpmainUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
   <item>
    <widget class="QWidget" name="leftsec" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="left_label">
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;URW Bookman L&quot;;</string>
        </property>
        <property name="text">
         <string>Boards</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="left_list">
        <property name="styleSheet">
         <string notr="true">font: 57 16pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="autoScrollMargin">
         <number>16</number>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <property name="text">
          <string>a1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>b2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>c3</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightsec" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="right_top" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deletepost">
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(239, 41, 41);</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectpost">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="postbut">
           <property name="text">
            <string>Post</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="right_list">
        <property name="styleSheet">
         <string notr="true">font: 75 oblique 16pt &quot;Waree&quot;;

</string>
        </property>
        <property name="spacing">
         <number>14</number>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
